### PR TITLE
Game creator

### PR DESCRIPTION
**Pull request by V-Fries**

### CODE REVIEW

File: .github/workflows/pong-server.yaml [hunk 1]
```diff
@@ -34,10 +34,18 @@ jobs:
...
 
-            - name: Run unit tests
+            - name: Copy shared files
+              run: |
+                  cp -r pong_server/src/shared_code pong_server/src/game_creator/shared_code
+
+            - name: Run unit tests game creator
               working-directory: pong_server/
               run: |
-                  python3 -m unittest discover test
```
Comment (tdameros) [inline .github/workflows/pong-server.yaml:40]: Why do you remove unittest in pong_server?

File: pong_server/src/game_creator/api/views.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        if players is None:
+            errors.append(error_messages.PLAYERS_FIELD_MISSING)
+            return []
+
+        if not isinstance(players, list):
+            errors.append(error_messages.PLAYERS_FIELD_IS_NOT_A_LIST)
+            return []
+
+        correct_players: set[int] = set()
+        for i in range(len(players)):
```
Comment (tdameros) [inline pong_server/src/game_creator/api/views.py:77]: You can use `enumerate` for iterate in players with indexes

**Review by tdameros — APPROVED** [review]

File: pong_server/src/game_creator/game_creator/settings.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+# Database
+# https://docs.djangoproject.com/en/4.2/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': BASE_DIR / 'db.sqlite3',
+    }
+}
```
Comment (EthanDelage) [inline pong_server/src/game_creator/game_creator/settings.py:85]: Create a condition to switch between sqlite3 (in DEBUG) and postgresql (in PROD)

**Review by EthanDelage — APPROVED** [review]

File: .github/workflows/pong-server.yaml (hunk 1 above)
Comment (V-Fries) [reply .github/workflows/pong-server.yaml:40]: The only tests that were written for pong server were for the redirection server which is now deprecated

File: pong_server/src/game_creator/game_creator/settings.py (hunk 3 above)
Comment (V-Fries) [reply pong_server/src/game_creator/game_creator/settings.py]: https://github.com/tdameros/42-transcendence/pull/182/commits/a3383ff2c1456907371830591a3f536df355dc81

File: pong_server/src/game_creator/api/views.py (hunk 2 above)
Comment (V-Fries) [reply pong_server/src/game_creator/api/views.py]: https://github.com/tdameros/42-transcendence/pull/182/commits/ee5528f7f554ea2552bd16676039825f81ce7071

**Pull request merged**